pkgs/build-support/setup-hooks/multiple-outputs.sh

```diff
@@ -154,6 +154,16 @@ _multioutDocs() {
 
     moveToOutput share/info "${!outputInfo}"
     moveToOutput share/doc "${!outputDoc}"
+
+    # move gi-docgen docs to devdoc
+    for output in $(getAllOutputNames); do
+        if [ -d "${!output}/share/doc" ]; then
+            while IFS= read -r -d '' dir; do
+                moveToOutput "share/doc/$(basename "$dir")" "${!outputDevdoc}"
+            done <  <(find "${!output}/share/doc" -type f -name "*devhelp*" -exec dirname "{}" \;)
```

Comment:
Also might be safer to use `*.devhelp2` as the pattern, gi-docgen does not generate older index versions.